quipucords/quipucords
# [DISCOVERY-193] Add Scan "name" property to OpenApi schema

## How to test

1. Spin up a swagger/api/mock container with this schema 
    `docker run -d --rm -p <PORT>:8000 -v "<PATH/TO/QUIPUCORDS>/docs/swagger.yml:/data/swagger.yaml"`
2. Hit /api/v1/scans/ and confirm if "name" property is reported
    `curl http://localhost:5000/api/v1/scans/ | jq '.results[].name'`

Jira issue: [DISCOVERY-193](https://issues.redhat.com/browse/DISCOVERY-193)
